xbmc/interfaces/legacy/Control.h

```diff
@@ -1473,15 +1473,16 @@ namespace XBMCAddon
     /**
      * ControlSlider class.
      * 
-     * ControlSlider(x, y, width, height[, textureback, texture, texturefocus])
+     * ControlSlider(x, y, width, height[, textureback, texture, texturefocus, orientation])
      * 
      * x              : integer - x coordinate of control.\n
      * y              : integer - y coordinate of control.\n
      * width          : integer - width of control.\n
      * height         : integer - height of control.\n
      * textureback    : [opt] string - image filename.\n
      * texture        : [opt] string - image filename.\n
-     * texturefocus   : [opt] string - image filename.n"
+     * texturefocus   : [opt] string - image filename.\n
+     * orientation    : [opt] integer - orientation of slider (0=horizontal / 1=vertical(default)).
```

Comment:
All you have to do is add

 ````
SWIG_CONSTANT(int, HORIZONTAL);
SWIG_CONSTANT(int, VERTICAL);
````
in ModuleXbmcgui.h. In python people can then write `ControlSlider(.. orientation=xbmcgui.VERTICAL)` instead of `orientation=0` and trying to remember what 0 means.